common/app/conf/switches.scala

```diff
@@ -228,6 +228,12 @@ object Switches {
     safeState = On, sellByDate = never
   )
 
+  val LiveBlogContributorImagesSwitch = Switch("Feature", "liveblog-contributor-image",
+    "Show contributor byline images in live blog blocks",
+    safeState = On,
```

Comment:
Probably should be `Off` to begin with. Then when this expires, we'll switch it to `On` and set the sellByDate to never.

I think the boxes tend to load the switches pretty sharpish when they start up so you probably wouldn't see this often (if at all).